cloudinary/utils.py

```diff
@@ -1264,6 +1264,56 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def compute_hex_hash(s):
```

Comment:
When you add a helper method, please update all code with the new method, search for `hashlib.sha1` in the project and replace with this function (where applicable)